groups/prometheus/module-ec2-instances/main.tf

```diff
@@ -0,0 +1,40 @@
+# EC2 Instance
```

Comment:
Remove the comment here and similarly the `# Outputs` comment later in this file—the resources are descriptive enough and comments should be limited to documenting justification for implementation details or known issues if anything.